package indigo.pb;

import "indigo_message.proto";

option java_package = "ru.indigosystem.serv.pb";

option java_outer_classname = "IndigoPB";
option optimize_for = LITE_RUNTIME;

message io_message {
    required uint64 content = 4;
    optional int32 content_size = 5;

    extend internal_msg {
            optional io_message io_message_in = 206;
    }

    extend indigo_msg {
            optional io_message io_message_out = 207;
    }
}

